src/rules/accessibility/checkbox-accessibility.js

```diff
@@ -0,0 +1,75 @@
+HTMLHint.addRule({
+    id: 'checkbox-acceessibility',
+    description: 'verify checkbox group container has role and aria-labelledby attributes, and checkbox input has accessibility bindings.',
+    init: function(parser, reporter){
+        var unclosedDivsCounter = 0;
+        var inCheckboxGroup = false;
+        var prevEvent;
+        var self = this;   
+        
+        var isCheckboxContainerWithoutRoleAttribute = function(event){
+            let roleAttribute = HTMLHint.utils.getAttributeValue(event.attrs,"role");
+            return isCheckboxGroupContainer(event) && roleAttribute !== 'group';
+         };
+
+        var isCheckboxContainerWithoutAriaLabelledAttribute = function(event){
+            let ariaLabelledbyAttribute = HTMLHint.utils.getAttributeValue(event.attrs,"aria-labelledby");        
```

Comment:
so if we want to keep consistent - it's better to replace let with var ?  (because all code until now uses var..)